utils/api.ts

```diff
@@ -187,10 +187,19 @@ export const initApi = async (uri = '') => {
       Bloom: {
         _: '[u8; 256]',
       },
+      BalanceLock: {
+        id: "[u8; 8]",
+        amount: 'Balance'
+      }
     },
   })
   // console.log(api.genesisHash.toHex())
   return api
 }
 
 
+//TODO: Remove when type mess is solved.
+export type BalanceLock = {
+  id: string,
+  amount: string
+}
```

Comment:
Was leaving this here intentional? Did the polkadot apps docs not help?

Reply:
Not sure why this is required. When you use .toHuman() does it not auto format it this way?

Reply:
I'll check the getters you mentioned and be back to this PR.

Reply:
removed!